### DISCUSSION TITLE
lotus-miner failed

### DISCUSSION THREAD

**Discussion by AYDIN23** [filecoin-project/lotus#9442] [General Q&A]
I am in the stage of sector pledge and I made 19 sectors until now. 
I had to format and re-partition one of the NVME drives to create a swap partition. 
Now when I run Lotus-miner I get error.

ERROR: creating node: starting node: could not build arguments for function "github.com/filecoin-project/lotus/node/modules".HandleDeals (/home/aydin/lotus/node/modules/storageminer.go:308): failed to build storagemarket.StorageProvider: could not build arguments for function "reflect".makeFuncStub (/usr/local/go/src/reflect/asm_amd64.s:28): failed to build *storedask.StoredAsk: could not build arguments for function "reflect".makeFuncStub (/usr/local/go/src/reflect/asm_amd64.s:28): failed to build storagemarket.StorageProviderNode: could not build arguments for function "reflect".makeFuncStub (/usr/local/go/src/reflect/asm_amd64.s:28): failed to build *sectorblocks.SectorBlocks: could not build arguments for function "reflect".makeFuncStub (/usr/local/go/src/reflect/asm_amd64.s:28): failed to build sectorblocks.SectorBuilder: could not build arguments for function "reflect".makeFuncStub (/usr/local/go/src/reflect/asm_amd64.s:28): failed to build *storage.Miner: could not build arguments for function "reflect".makeFuncStub (/usr/local/go/src/reflect/asm_amd64.s:28): failed to build sectorstorage.SectorManager: could not build arguments for function "reflect".makeFuncStub (/usr/local/go/src/reflect/asm_amd64.s:28): failed to build *sectorstorage.Manager: could not build arguments for function "reflect".makeFuncStub (/usr/local/go/src/reflect/asm_amd64.s:28): failed to build *stores.Local: received non-nil error from function "reflect".makeFuncStub (/usr/local/go/src/reflect/asm_amd64.s:28): opening path : unmarshalling storage metadata for : invalid character '"' after object key:value pair

I dont know what to do.

**Comment by Reiers** [comment]
Hi @AYDIN23 

What was on the NVMe disks before you formatted them?
- `.lotusminer` path?
- `.lotus` path?

Or was it just sealing storage/space?

**Reply by AYDIN23** [reply]
there were 7 files named : cache, sealed, unsealed, update, update-cache, trash-1000 and sectorstore.json

**Comment by Reiers** [comment]
Please check the `storage.json` in `.lotusminer` directory, it might be because you have the old path in your `storage.json` file. 
I assume that you need to update the path in order for it to start working again.

**Reply by AYDIN23** [reply]
I checked. There was no problem.
now when i run lotus-miner:
24.47 GiB / 56.61 GiB [============================>--------------------------------------]  43.24% 3.85 MiB/s 2h22m36s2022-10-08T04:42:29.921+0330	INFO	paramfetch	go-paramfetch@v0.0.4/paramfetch.go:209	Parameter file /var/tmp/filecoin-proof-parameters/v28-stacked-proof-of-replication-merkletree-poseidon_hasher-8-8-0-sha256_hasher-82a357d2f2ca81dc61bb45f4a762807aedee1b0a53fd6c4e77b46a01bfef7820.params is ok
